app/forms/form_answer.rb

```diff
@@ -0,0 +1,16 @@
+class FormAnswer
```

Comment:
**nit** if we use a separate class, rather than struct this a. should have a spec b. probably better named feedback_form_answer, or namespaced as it is quite specific to the feedback form currently. 

I personally think a Struct would be more suitable and implicitly handle the naming/namespacing and testing grey areas.

```
class FeedbackForm
  Answer = Struct.new(:key, :label, :id, :other) do
    def initialize(:key, :label, :id, other: false)
      super
    end
    
    def id
      return { id: id, other: true } if other
      id
    end
  end
 ...
end
```

Reply:
I'll move it back into a struct. I had a thought about going further with this but it was getting overly complicated.